nengo_viz/static/viz_netgraph.js

```diff
@@ -8,9 +8,11 @@
  */
 VIZ.NetGraph = function(parent, args) {
     this.scale = 1.0;          // global scaling factor
-    this.offsetX = 0;          // global x,y pan offset 
+    this.offsetX = 0;          // global x,y pan offset
     this.offsetY = 0;
-    
+    this.zoom_fonts = false;    // scale fonts when zooming
+    this.font_size = 100;       // font size as a percent of base
```

Comment:
[Here the unit was also documented and it still went wrong.](https://en.wikipedia.org/wiki/Mars_Climate_Orbiter) Good thing we're not building space crafts. :rocket: :boom: